### PR TITLE
act: fix error for running job

### PR DESCRIPTION
**Pull request by iTrooz**
- [x] The page(s) have at most 8 examples.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 0.2.57

Fix https://github.com/tldr-pages/tldr/issues/12116

### CODE REVIEW

File: pages.fr/common/act.md [hunk 1]
```diff
@@ -15,9 +15,9 @@
 
 `act {{type_d_événement}}`
 
-- Execute une action spécifique :
+- Execute un job spécifique :
```
Comment (vitorhcl) [inline pages.fr/common/act.md:18]:
```suggestion
- Execute un [j]ob spécifique :
```
We can add this mnemonic too

Comment (iTrooz) [reply pages.fr/common/act.md]: Done

**Comment by iTrooz** [comment]
Thanks !

**Comment by vitorhcl** [comment]
If you have the time, could you please update the French page to match the English version?

**Comment by iTrooz** [comment]
No problem ! done

File: pages/common/act.md [hunk 2]
```diff
@@ -27,6 +27,6 @@
 
 `act -v`
 
-- Run a specific [W]orkflow:
+- Run a specific [W]orkflow with the "push" event:
```
Comment (vitorhcl) [inline pages/common/act.md:30]:
```suggestion
- Run a specific [W]orkflow with the push event:
```

File: pages.fr/common/act.md [hunk 3]
```diff
@@ -26,3 +26,7 @@
 - Affiche le journal en mode verbeux :
 
 `act -v`
+
+- Execute un [W]orkflow en particulier, avec l'événement "push":
```
Comment (vitorhcl) [inline pages.fr/common/act.md:30]:
```suggestion
- Execute un [W]orkflow en particulier, avec l'événement push:
```
I think we don't need the double quotes

**Review by vitorhcl — APPROVED** [review]
Thanks for your fast response and contribution! I have a minor suggestion

File: pages.fr/common/act.md (hunk 3 above)
Comment (iTrooz) [reply pages.fr/common/act.md]: Done !

File: pages.fr/common/act.md [hunk 4]
```diff
@@ -26,3 +26,7 @@
 - Affiche le journal en mode verbeux :
 
 `act -v`
+
+- Execute un [W]orkflow en particulier, avec l'événement push:
```
[the remark this answers is not in the document]
Comment (iTrooz) [reply pages.fr/common/act.md]: Done

**Review by kbdharun — APPROVED** [review]

**Review by Nico385412 — APPROVED** [review]

**Pull request merged**